src/react-apps/applications/ux-editor/src/containers/WorkflowStep.tsx

```diff
@@ -63,57 +102,103 @@ class WorkflowStepComponent extends React.Component<IWorkflowStepProps, any> {
   public renderNavBar = () => {
     return (
       <div className='a-modal-navbar'>
-        <button type='button' className='a-modal-back a-js-tabable-popover'aria-label='Tilbake'>
+        {this.props.step === WorkflowSteps.FormFilling &&
+          <button type='button' className='a-modal-back a-js-tabable-popover' aria-label='Tilbake'>
+            <span className='ai-stack'>
+              <i className='ai ai-stack-1x ai-plain-circle-big' aria-hidden='true' />
+              <i className='ai-stack-1x ai ai-back' aria-hidden='true' />
+            </span>
+          </button>
+        }
+        <button
+          type='button'
+          className='a-modal-close a-js-tabable-popover'
+          aria-label='Lukk'
+          onClick={this.handleClose}
+        >
           <span className='ai-stack'>
-            <i className='ai ai-stack-1x ai-plain-circle-big' aria-hidden='true'/>
-            <i className='ai-stack-1x ai ai-back' aria-hidden='true'/>
-          </span>
-        </button>
-        <button type='button' className='a-modal-close a-js-tabable-popover' aria-label='Lukk'>
-          <span className='ai-stack'>
-            <i className='ai ai-stack-1x ai-plain-circle-big' aria-hidden='true'/>
-            <i className='ai-stack-1x ai ai-exit  a-modal-close-icon' aria-hidden='true'/>
+            <i className='ai ai-stack-1x ai-plain-circle-big' aria-hidden='true' />
+            <i className='ai-stack-1x ai ai-exit  a-modal-close-icon' aria-hidden='true' />
           </span>
         </button>
       </div>
     );
   }
 
+  public handleClose = () => {
+    this.setState({
+      redirect: true,
+    });
+  }
+
+  public handleSubmitForm = () => {
+    const altinnWindow: IAltinnWindow = window as IAltinnWindow;
+    const { org, service, instanceId } = altinnWindow;
+    if (window.location.pathname.split('/')[1].toLowerCase() === 'runtime') {
+      FormFillerActionDispatchers.completeAndSendInForm(
+        `${window.location.origin}/runtime/${org}/${service}/${instanceId}/CompleteAndSendIn`);
+    }
+  }
+  public renderFormFiller = () => {
+    return this.props.children;
+  }
+
+  public renderSubmit(): React.ReactNode {
+    return (
+      <button
+        type='submit'
+        className={'a-btn a-btn-success'}
+        onClick={this.handleSubmitForm}
+        id='workflowSubmitStepButton'
+      >
+        Send Inn
```

Comment:
Text should be in language files, under form_filler section

Reply:
Fixed.